app/core/Engine.ts

```diff
@@ -350,6 +358,9 @@ class Engine {
   snapExecutionService: WebViewExecutionService;
   ///: END:ONLY_INCLUDE_IF
 
+  txController: TransactionController;
+  stxController: SmartTransactionsController;
```

Comment:
```suggestion
  transactionController: TransactionController;
  smartTransactionController: SmartTransactionsController;
```
Apart from style, it's a bit too easy to glance over a typo with these otherwise.